### ISSUE TITLE
feat: Unit tests

### ISSUE THREAD

**Issue by kristovatlas** [0xd34db33f/gfyp#6]
Some unit tests would be helpful. This is easiest with database-related stuff; network behavior is slower and thus less convenient to unit test.

**Comment by 0xd34db33f** [comment]
Yeah definitely makes sense to add some good testing functionality to it. Thinking out loud really as I'm not sure where the testing segmentation lies between the wrapping code vs dnstwist code. If we end up testing the dnstwist functionality, wonder if it makes sense pushing something upstream and then modeling the additional tests we are building for GFYP based on that. I'll muse on it here.
